cg/meta/workflow/nf_analysis.py

```diff
@@ -56,6 +56,12 @@ def process(self):
     def process(self, process: Process):
         self._process = process
 
+    @property
+    def threshold_reads(self):
```

Comment:
```suggestion
    def pass_reads_threshold(self) -> bool:
```

Reply:
I will not do that here. This is the name required by analysis api and the name used in all other pipelines

Reply:
well. ended up doing it here as Henrik also had the same comment